### PR TITLE
Minor Test Improvements

### PR DESCRIPTION
**Pull request by luis-cortes**
* Fixed an accidentally duplicated test
* Improved error messaging in a few different tests.

### CODE REVIEW

File: picasso/src/test/java/com/squareup/picasso3/BitmapHunterTest.kt [hunk 1]
```diff
@@ -165,14 +165,11 @@ class BitmapHunterTest {
     assertThat(eventRecorder.decodedBitmap).isNull()
   }
 
-  @Test fun huntUnrecognizedUri() {
+  @Test(expected = IllegalStateException::class)
```
Comment (JakeWharton) [inline picasso/src/test/java/com/squareup/picasso3/BitmapHunterTest.kt:168]: Please never use this. An earlier statement in the body could throw and your test would still pass.

File: picasso/src/test/java/com/squareup/picasso3/BitmapHunterTest.kt [hunk 2]
```diff
@@ -971,15 +968,13 @@ class BitmapHunterTest {
     val original = android.graphics.Bitmap.createBitmap(10, 10, ARGB_8888)
     val result = RequestHandler.Result.Bitmap(original, MEMORY, 0)
     val data = Request.Builder(URI_1).build()
-    try {
+    val exception = assertThrows(RuntimeException::class.java) {
```
Comment (JakeWharton) [inline picasso/src/test/java/com/squareup/picasso3/BitmapHunterTest.kt:971]: kotlin.test.assertFailsWith reads nicer in Kotlin sources

File: picasso/src/test/java/com/squareup/picasso3/BitmapHunterTest.kt (hunk 1 above)
Comment (luis-cortes) [reply picasso/src/test/java/com/squareup/picasso3/BitmapHunterTest.kt:168]: Ah fair enough. Reverting!

File: picasso/src/test/java/com/squareup/picasso3/RequestCreatorTest.kt [hunk 3]
```diff
@@ -557,7 +557,7 @@ class RequestCreatorTest {
 
   @Test fun invalidCenterInside() {
     try {
-      mockRequestCreator(picasso).resize(10, 10).centerInside().centerCrop()
+      mockRequestCreator(picasso).resize(10, 10).centerCrop().centerInside()
```
Comment (jrodbx) [inline picasso/src/test/java/com/squareup/picasso3/RequestCreatorTest.kt:560]: oh wow, good eye

**Review by jrodbx — APPROVED** [review]

**Pull request merged**